solr/core/src/java/org/apache/solr/rest/RestManager.java

```diff
@@ -355,10 +355,7 @@ public void delegateRequestToManagedResource() {
     }
 
     private void doHead(ManagedEndpoint managedEndpoint) {
-      // Setting the response to blank clears the content out,
-      // however it also means the Content-Length HTTP Header is set to 0
-      // which isn't compliant with the specification of how HEAD should work,
-      // it should instead return the length of the content if you did a GET.
```

Comment:
This is an incorrect statement about head request and `Content-Length` see https://www.rfc-editor.org/rfc/rfc9110.html#name-head